apps/epic-react/src/pages/invoices/[merchantChargeId].tsx

```diff
@@ -90,6 +103,21 @@ const Invoice: React.FC<
       className="print:bg-white print:text-black"
     >
       <main className="mx-auto max-w-screen-md">
+        {purchaseDetails && purchaseDetails.purchase && (
+          <Card
+            title={{as: 'h1', content: 'Invite your team'}}
+            icon={
+              <UserGroupIcon className="w-5 text-cyan-500" aria-hidden="true" />
+            }
+          >
+            <InviteTeam
+              session={session}
+              purchase={purchaseDetails.purchase}
+              existingPurchase={purchaseDetails.existingPurchase}
+              setPersonalPurchase={() => {}}
+            />
+          </Card>
+        )}
```

Comment:
the current link on the home page in the WelcomeBanner links directly to an invoice so i plopped this in here